src/components/Editor/SearchBar.js

```diff
@@ -280,13 +280,16 @@ class SearchBar extends Component<Props, State> {
     );
   };
 
+  shouldShowErrorEmoji() {
+    return true;
```

Comment:
why do we always want to show the error?

Reply:
There are three conditions for the error emoji to appear - this flag passed into SearchInput + count being 0 + search input not being empty. if we don't set should `shouldShowErrorEmoji` to true the sad face will never show since the flag will always exit on false.